src/worker/core/GameSim.basketball.ts

```diff
@@ -271,7 +271,13 @@ class GameSim {
 			for (let p = 0; p < this.team[t].player.length; p++) {
 				for (const r of Object.keys(this.team[t].player[p].compositeRating)) {
 					if (r !== "endurance") {
-						this.team[t].player[p].compositeRating[r] *= factor;
```

Comment:
If including this, we need to heck and see if the default home court advantage magnitude needs to be adjusted.